lib/pages/home.dart

```diff
@@ -49,7 +52,23 @@ class _HomeState extends State<Home> {
               iconSize: 42.0
             ),
           ],
-        ),],
+        ),
```

Comment:
please remove button or create mock data

Reply:
this will now contain a mock route to provide the possibility to test share functionality for Route Object integration